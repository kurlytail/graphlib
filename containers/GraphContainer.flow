<?xml version="1.0" encoding="UTF-8"?>
<xmi:XMI xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:flow="http://www.brainspeedtech.com/swarm/flow" xmlns:notation="http://www.eclipse.org/gmf/runtime/1.0.2/notation">
  <flow:Flow xmi:id="_ADNhUEXCEemKT5rL-yWemg" name="GraphContainer"/>
  <notation:Diagram xmi:id="_ADOIYEXCEemKT5rL-yWemg" type="Flow" element="_ADNhUEXCEemKT5rL-yWemg" name="GraphContainer.flow" measurementUnit="Pixel">
    <styles xmi:type="notation:DiagramStyle" xmi:id="_ADOIYUXCEemKT5rL-yWemg"/>
  </notation:Diagram>
</xmi:XMI>
