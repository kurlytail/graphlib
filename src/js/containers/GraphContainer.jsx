import { ButtonComponent } from '../components';
import React, { Component } from 'react';

class GraphContainer extends Component {
    render() {
        const Container = (
            <div className={'container-fluid'}>
                <ButtonComponent label="rebuild" />
            </div>
        );
        return Container;
    }
}

export default GraphContainer;
