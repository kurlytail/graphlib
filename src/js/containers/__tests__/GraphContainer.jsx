import React from 'react';
import Renderer from 'react-test-renderer';

import GraphContainer from '../GraphContainer';

describe('# GraphContainer', () => {
    describe('## constructor', () => {
        it('### should create default object', () => {
            const container = Renderer.create(<GraphContainer />);
            expect(container.toJSON()).toMatchSnapshot();
        });
    });
});
