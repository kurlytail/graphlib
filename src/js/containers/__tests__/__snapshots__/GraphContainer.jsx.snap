// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`# GraphContainer ## constructor ### should create default object 1`] = `<div />`;
